frontend/packages/dev-console/src/components/add/AddPageLayout.scss

```diff
@@ -39,8 +41,10 @@
         display: flex;
       }
       &__text {
-        margin-left: var(--pf-global--spacer--xs);
-        vertical-align: middle;
+        // Used font-seze-base variable instead of --pf-global--FontSize--sm, which is getting calculated to 13px instead of 14px
```

Comment:
Typo:
```suggestion
        // Used font-size-base variable instead of --pf-global--FontSize--sm, which is getting calculated to 13px instead of 14px
```

Reply:
Corrected the typo in comment.